docs/README.md

```diff
@@ -34,7 +34,7 @@ To install the latest version of the `gaia` application and join the public test
 
 ## Contribute
 
-See [this file](./DOCS_README.md) for details of the build process and
+See [this file](https://github.com/cosmos/cosmos-sdk/blob/master/docs/DOCS_README.md) for details of the build process and
```

Comment:
should we link to the `develop` branch, since that's what we're defaulting to on Github?

Reply:
we use master branch for links for everything else tho

Reply:
also, same PR in tendermint uses master, not sure why we'd diverge

Reply:
Tendermint has `master` as the default branch though

Reply:
but not a big deal